pkg/libvmi/cloudinit/cloudinit.go

```diff
@@ -38,3 +38,15 @@ func WithNoCloudEncodedUserData(data string) NoCloudOption {
 		source.UserDataBase64 = base64.StdEncoding.EncodeToString([]byte(data))
 	}
 }
+
+func WithNoCloudNetworkData(data string) NoCloudOption {
+	return func(source *v1.CloudInitNoCloudSource) {
+		source.NetworkData = data
+	}
+}
+
+func WithNoCloudEncodedNetworkData(data string) NoCloudOption {
+	return func(source *v1.CloudInitNoCloudSource) {
+		source.NetworkDataBase64 = base64.StdEncoding.EncodeToString([]byte(data))
+	}
```

Comment:
This needs to move to the next commit.

Reply:
reminding you that you didnt move this to the next commit :)

Reply:
Done